models/Asset/Thumbnail/ImageThumbnailTrait.php

```diff
@@ -429,4 +430,24 @@ private function getFilename(): string
 
         return basename($pathReference['src']);
     }
+
+
+    /**
+     * @description Returns path for thumbnail image in a given file format
+     * @param string $format
+     * @return self
+     */
+    public function getAsFormat(string $format): self
+    {
+        $config = $this->getConfig();
+
+        if (!$config) {
+            $config = new Config();
+        }
```

Comment:
```suggestion
        $config = $this->getConfig() ?? new Config();
```

Reply:
Don't you want to change this? The `@description` tag isn't used anywhere else in Pimcore and I think the `@return $this` is more correct if you `return $this;`.